core/src/test/java/apoc/meta/MetaTest.java

```diff
@@ -171,91 +167,6 @@ public void testMetaGraphExtraRels() throws Exception {
-    @Test
-    public void testMetaTypeArray() throws Exception {
-        testTypeName(asList(1,2), "LIST");
-        testTypeName(asList(LocalDate.of(2018, 1, 1),2), "LIST");
-        testTypeName(new Integer[] {1, 2}, "int[]");
-        testTypeName(new Float[] {1f, 2f}, "float[]");
-        testTypeName(new Double[] {1d, 2d}, "double[]");
-        testTypeName(new String[] {"a", "b"}, "String[]");
-        testTypeName(new Long[] {1l, 2l}, "long[]");
-        testTypeName(new LocalDate[] {LocalDate.of(2018, 1, 1), LocalDate.of(2018, 1, 1)}, "LIST");
-        testTypeName(new Object[] {1d, ""}, "LIST");
-    }
-
-    @Test
-    public void testMetaIsType() throws Exception {
-        try (Transaction tx = db.beginTx()) {
-            Node node = tx.createNode();
-            Relationship rel = node.createRelationshipTo(node, RelationshipType.withName("FOO"));
-            testIsTypeName(node, "NODE");
-            testIsTypeName(rel, "RELATIONSHIP");
-            Path path = tx.traversalDescription().evaluator(toDepth(1)).traverse(node).iterator().next();
-// TODO PATH FAILS            testIsTypeName(path, "PATH");
-            tx.rollback();
-        }
-        testIsTypeName(singletonMap("a", 10), "MAP");
-        testIsTypeName(asList(1, 2), "LIST");
-        testIsTypeName(1L, "INTEGER");
-        testIsTypeName(1, "INTEGER");
-        testIsTypeName(1.0D, "FLOAT");
-        testIsTypeName(1.0, "FLOAT");
-        testIsTypeName("a", "STRING");
-        testIsTypeName(false, "BOOLEAN");
-        testIsTypeName(true, "BOOLEAN");
-        testIsTypeName(null, "NULL");
-    }
-    @Test
-    public void testMetaTypes() throws Exception {
-
-        Map<String, Object> param = map("MAP", singletonMap("a", 10),
-                "LIST", asList(1, 2),
-                "INTEGER", 1L,
-                "FLOAT", 1.0D,
-                "STRING", "a",
-                "BOOLEAN", true,
-                "NULL", null);
-        TestUtil.testCall(db, "RETURN apoc.meta.types($param) AS value", singletonMap("param",param), row -> {
-            Map<String,String> res = (Map) row.get("value");
-            res.forEach(Assert::assertEquals);
-        });
-
-    }
-
-    private void testTypeName(Object value, String type) {
-        TestUtil.testCall(db, "RETURN apoc.meta.typeName($value) AS value", singletonMap("value", value), row -> assertEquals(type, row.get("value")));
-//        TestUtil.testCall(db, "RETURN apoc.meta.type($value) AS value", singletonMap("value", value), row -> assertEquals(type, row.get("value")));
-    }
-
-    private void testIsTypeName(Object value, String type) {
-        TestUtil.testCall(db, "RETURN apoc.meta.isType($value,$type) AS value", map("value", value, "type", type), result -> assertEquals("type was not "+type,true, result.get("value")));
-        TestUtil.testCall(db, "RETURN apoc.meta.isType($value,$type) AS value", map("value", value, "type", type + "foo"), result -> assertEquals(false, result.get("value")));
-    }
-
```

Comment:
Many of the apoc.meta functions have been simplified to apoc.meta.type, but apoc.meta.type currently only has two tests which tests Point and Duration data types. No real idea what this does but it seems like it used to test many different types, and it would perhaps be worth converting these tests rather than deleting them outright.